plugin.json

```diff
@@ -43,7 +43,7 @@
                 "key": "EncryptionSecret",
                 "display_name": "Encryption Secret:",
                 "type": "generated",
-                "help_text": "The secret key is used to encrypt and decrypt the OAuth token.\nRegenerating the secret key will cause the loss of data."
+                "help_text": "The secret key is used to encrypt and decrypt the OAuth token.\nRegenerating the secret will require all users to re-connect their accounts to Azure DevOps."
```

Comment:
```suggestion
                "help_text": "The secret key used to encrypt and decrypt the OAuth token.\nRegenerating the secret will require all users to re-connect their accounts to Azure DevOps."
```

Reply:
I guess this already looks fine.

Reply:
when we are writing `is` here the meaning is changed 

as by reading `The secret key is used to encrypt and decrypt the OAuth token` it seems like we are telling user what is a `secret key`

but our goal is not that here we are not explaining or describing the secret key itself but we are just saying that this the secret key which is used for encrypt/decrypt

so reading `The secret key used to encrypt and decrypt the OAuth token` just tells what we need

Reply:
okay